style-sheets/primevue-components-overrides.scss

```diff
@@ -1203,6 +1192,14 @@ label {
     border-left: none;
 }
 
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:first-child
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-top-left-radius: 0.25rem;
+    border-left: solid 0.0625rem map.get(lightTheme.$light-theme, 'border-subtle-00');
+}
+
```

Comment:
had to move this code I had to move further down because when the second tab is hidden the style did no apply

Reply:
good catch!